src/components/atoms/Button/Button.styled.ts

```diff
@@ -46,24 +46,28 @@ const setIconPosition = (pos: IconPosition) => {
     default:
       return '0';
   }
-}
+};
 
 const setBackgroundColor = (type: ButtonType): string => {
   switch (type) {
-    case 'fill': return colors.background.NeutralWhite;
-    case 'outline': return colors.background.Primary40;
-    case 'transparent': return colors.background.Primary40;
-    default: return colors.background.Neutral40;
+    case 'fill':
+      return colors.background.NeutralWhite;
+    case 'outline':
+      return colors.background.Primary40;
+    case 'transparent':
+      return colors.background.Primary40;
+    default:
+      return colors.background.Neutral40;
   }
-}
+};
 
-export const ButtonIcon = styled.div<{ type: ButtonType, iconPos: IconPosition }>`
+export const ButtonIcon = styled.div<{ type: ButtonType; iconPos: IconPosition }>`
```

Comment:
Are you sure it should be `;` instead of `,` ?

Reply:
That was hint from eslint or prettier

Reply:
@hendrysiak is this correct?